houston/mutations.go

```diff
@@ -0,0 +1,30 @@
+package houston
```

Comment:
please move to  https://github.com/astronomer/astro-cli/blob/8ee42debef6220dea7204bac9118d74eb094a30d/houston/queries.go#L1

Reply:
I saw there are mutations in that `queries.go` file, but is it worth separating those out like we do in Houston API into `queries.go`, `mutations.go`, `subscriptions.go`?  If so, I was going to make a ticket to re-organize the existing mutations and subscriptions that are inside of the `queries.go` file.

Reply:
I added a TODO for this issue.  We can decide how to reorganize the queries: https://github.com/astronomer/issues/issues/1991